bolt-service/scripts/setup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+BOLT_SERVER_PACKAGE=pe-bolt-server_0.22.0.7.g3257f59-1bionic_amd64.deb
```

Comment:
Does this get updated somehow? Should we read it from a file instead, so the file can be updated as we release new versions? I can also talk to RE about making a 'latest' directory, though they likely have higher priorities for the next few weeks.